### PR TITLE
Karmyzov  lesson 04

### LINKED ISSUE #40: Karmyzov: lesson04

### PR DESCRIPTION
**Pull request by skarmyzov**
close #40

### CODE REVIEW

File: karmyzov/READMI.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+= Karmyzov
```
Comment (rakovets) [inline karmyzov/READMI.adoc:1]: Filename is wrong

File: karmyzov/lesson04/resume/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Project exclude paths
+/target/
```
Comment (rakovets) [inline karmyzov/lesson04/resume/.gitignore:2]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

File: karmyzov/lesson04/resume/pom.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>3.3.2</version>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (rakovets) [inline karmyzov/lesson04/resume/pom.xml:50]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

File: karmyzov/lesson04/resume/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="https://jakarta.ee/xml/ns/jakartaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd"
+         version="5.0">
+</web-app>
```
Comment (rakovets) [inline karmyzov/lesson04/resume/src/main/webapp/WEB-INF/web.xml:6]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

File: karmyzov/lesson04/resume/src/main/webapp/index.jsp [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+</main>
+
+<footer>
+    <p>
+        (c) Sergei Karmyzov - 2021
+    </p>
+</footer>
+</body>
+
+</html>
```
Comment (rakovets) [inline karmyzov/lesson04/resume/src/main/webapp/index.jsp:101]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix

File: karmyzov/lesson04/resume/src/main/webapp/index.jsp (hunk 5 above)
Comment (skarmyzov) [reply karmyzov/lesson04/resume/src/main/webapp/index.jsp]: resolve

File: karmyzov/READMI.adoc (hunk 1 above)
Comment (skarmyzov) [reply karmyzov/READMI.adoc]: Fixed

File: karmyzov/lesson04/resume/.gitignore (hunk 2 above)
Comment (skarmyzov) [reply karmyzov/lesson04/resume/.gitignore]: Добавляю строку  в Idea , когда делаю commit push в github все равное нет пустой строки

**Review by rakovets — APPROVED** [review]
Approved

**Pull request merged**